docs/en/compute/Hive.md

```diff
@@ -65,7 +65,7 @@ $ ./bin/alluxio fs mkdir /ml-100k
 $ ./bin/alluxio fs copyFromLocal /path/to/ml-100k/u.user alluxio://master_hostname:port/ml-100k
 ```
 
-View Alluxio WebUI at `http://master_hostname:port` and you can see the directory and file Hive
+View Alluxio WebUI at `http://master_hostname:19999` and you can see the directory and file Hive
```

Comment:
To avoid confusion with the RPC port which is used in Alluxio URIs throughout the whole doc.